packages/wdio-reporter/src/supportsColor.ts

```diff
@@ -8,14 +8,9 @@ import process from 'node:process'
 import os from 'node:os'
 import tty from 'node:tty'
 
-declare global {
-    // eslint-disable-next-line no-var
-    var Deno: any
-}
-
 // From: https://github.com/sindresorhus/has-flag/blob/main/index.js
 /// function hasFlag(flag, argv = globalThis.Deno?.args ?? process.argv) {
-function hasFlag(flag: string, argv = globalThis.Deno ? globalThis.Deno.args : process.argv) {
+function hasFlag(flag: string, argv = process.argv) {
```

Comment:
The Deno check has been removed, is this intentional?

Reply:
Yes, the assumption is that Deno now better supports Node.js primitives. Nonetheless, we haven't made any claims to support Deno.